CDK/Linux/hello-world/lib/aws-image-builder.ts

```diff
@@ -155,6 +159,22 @@ export class AWSImageBuilderConstruct extends Construct {
       version: props.version,
       parentImage: parentImageID,
       components: componentArn,
+      additionalInstanceConfiguration: {
+        systemsManagerAgent: {
+          uninstallAfterBuild: terminationConfig,
```

Comment:
Is there a reason why this sample would want to remove the SSM Agent, even if debug is set to false?

Reply:
you are right, thanks for picking this up, it was part of my manual testing to figure out a vague error message. removed this config.